assets/scripts/vendor.js

```diff
@@ -8,11 +8,16 @@ const btnMonthly = document.getElementById('monthly');
 const planValue = document.getElementById('plan-value');
```

Comment:
Could you split the logic for each car into different files?  One .js file for the plan-overview card and another js file for checkout-pop. The reason is to isolate each logic and also make code easier to read and maintain.